KsApi/MockService.swift

```diff
@@ -514,6 +515,19 @@ internal struct MockService: ServiceType {
     )
   }
 
+  func fetchGraphCategories(query: NonEmptySet<Query>) -> SignalProducer<RootCategoriesEnvelope, GraphError> {
+    return SignalProducer(value: self.fetchGraphCategoriesResponse!)
+  }
+
+  func fetchGraphCategory(query: NonEmptySet<Query>)
+    -> SignalProducer<RootCategoriesEnvelope.Category, GraphError> {
+    return SignalProducer(value: .template |> RootCategoriesEnvelope.Category.lens.id .~ "\(query.head)")
```

Comment:
This should be a property on the `MockService` so that we can control it from tests.